[2025-08-09T19:53:19.620+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-09T19:53:19.663+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xrs_update_12h.dq_quick manual__2025-08-09T19:53:05+00:00 [queued]>
[2025-08-09T19:53:19.672+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xrs_update_12h.dq_quick manual__2025-08-09T19:53:05+00:00 [queued]>
[2025-08-09T19:53:19.681+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-08-09T19:53:19.698+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dq_quick> on 2025-08-09 19:53:05+00:00
[2025-08-09T19:53:19.709+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'xrs_update_12h', 'dq_quick', 'manual__2025-08-09T19:53:05+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/xrs_update_daily.py', '--cfg-path', '/tmp/tmp1n2in7n3']
[2025-08-09T19:53:19.713+0000] {standard_task_runner.py:91} INFO - Job 68: Subtask dq_quick
[2025-08-09T19:53:19.713+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=300) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-09T19:53:19.714+0000] {standard_task_runner.py:63} INFO - Started process 301 to run task
[2025-08-09T19:53:19.785+0000] {task_command.py:426} INFO - Running <TaskInstance: xrs_update_12h.dq_quick manual__2025-08-09T19:53:05+00:00 [running]> on host 9d4e07229e62
[2025-08-09T19:53:19.914+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='xrs_update_12h' AIRFLOW_CTX_TASK_ID='dq_quick' AIRFLOW_CTX_EXECUTION_DATE='2025-08-09T19:53:05+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-09T19:53:05+00:00'
[2025-08-09T19:53:19.915+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-09T19:53:19.930+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-09T19:53:19.933+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "python - <<'PY'\nimport pandas as pd\ndf=pd.read_parquet('data/xrs_clean.parquet')\nprint('shape:', df.shape)\nprint('nulls:\\n', df.isna().sum().sort_values(ascending=False).to_string())\nprint('by satellite:\\n', df.groupby('satellite')['time'].agg(['min','max','count']))\nPY"]
[2025-08-09T19:53:19.949+0000] {subprocess.py:86} INFO - Output:
[2025-08-09T19:53:20.865+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2025-08-09T19:53:20.866+0000] {subprocess.py:93} INFO -   File "<stdin>", line 2, in <module>
[2025-08-09T19:53:20.866+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/pandas/io/parquet.py", line 670, in read_parquet
[2025-08-09T19:53:20.867+0000] {subprocess.py:93} INFO -     return impl.read(
[2025-08-09T19:53:20.868+0000] {subprocess.py:93} INFO -            ^^^^^^^^^^
[2025-08-09T19:53:20.868+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/pandas/io/parquet.py", line 265, in read
[2025-08-09T19:53:20.869+0000] {subprocess.py:93} INFO -     path_or_handle, handles, filesystem = _get_path_or_handle(
[2025-08-09T19:53:20.869+0000] {subprocess.py:93} INFO -                                           ^^^^^^^^^^^^^^^^^^^^
[2025-08-09T19:53:20.870+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/pandas/io/parquet.py", line 139, in _get_path_or_handle
[2025-08-09T19:53:20.870+0000] {subprocess.py:93} INFO -     handles = get_handle(
[2025-08-09T19:53:20.871+0000] {subprocess.py:93} INFO -               ^^^^^^^^^^^
[2025-08-09T19:53:20.871+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.12/site-packages/pandas/io/common.py", line 872, in get_handle
[2025-08-09T19:53:20.872+0000] {subprocess.py:93} INFO -     handle = open(handle, ioargs.mode)
[2025-08-09T19:53:20.872+0000] {subprocess.py:93} INFO -              ^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-08-09T19:53:20.873+0000] {subprocess.py:93} INFO - FileNotFoundError: [Errno 2] No such file or directory: 'data/xrs_clean.parquet'
[2025-08-09T19:53:21.022+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-08-09T19:53:21.023+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-09T19:53:21.035+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-08-09T19:53:21.040+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=xrs_update_12h, task_id=dq_quick, run_id=manual__2025-08-09T19:53:05+00:00, execution_date=20250809T195305, start_date=20250809T195319, end_date=20250809T195321
[2025-08-09T19:53:21.061+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 68 for task dq_quick (Bash command failed. The command returned a non-zero exit code 1.; 301)
[2025-08-09T19:53:21.094+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-08-09T19:53:21.117+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-09T19:53:21.128+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
